type-generation/README.md

```diff
@@ -0,0 +1,66 @@
+# GraphApi.io Type generation
+
+In this simple example we will show how to quickly generate typescript definitions for you Graph Api.
+
+## Steps
+
+### Clone this repository
+
+```
+git clone https://github.com/graphapi-io/resources.git
+cd resources
+```
+
+### Install dependencies
+
+```
+ npm i
+```
+
+### Get your api key
+
+1.  Go into your API Settings on https://my.graphapi.io
+1.  You may be asked to log in to your API. If you haven't done it yet, sign up using the same email that you use for signing into the my.graphapi.io
```

Comment:
better: sign up using the same email address you're signed in with on my.graphapi.io